js/kanban.js

```diff
@@ -97,46 +192,22 @@
       this.supported_itemtypes = {};
 
       /**
-       * If true, then a button will be added to each column to allow new items to be added.
-       * When an item is added, a request is made via AJAX to create the item in the DB.
-       * Permissions are re-checked server-side during this request.
-       * Users will still be limited by {@link limit_addcard_columns} both client-side and server-side.
-       * @since 9.5.0
-       * @type {boolean}
+       * User rights object
+       * @type {GLPIKanbanRights}
        */
-      this.allow_add_item = false;
+      this.rights = new GLPIKanbanRights({});
 
-      /**
-       * If true, then a button will be added to the add column form that lets the user create a new column.
-       * For Projects as an example, it would create a new project state.
-       * Permissions are re-checked server-side during this request.
-       * @since 9.5.0
-       * @type {boolean}
-       */
+      /** @deprecated x.x.x Use rights.canCreateItem() instead */
```

Comment:
Is this really required to keep that deprecated code? Does anything outside GLPI uses this object?

Reply:
I don't know for sure. There could be a plugin out there that uses them. It probably isn't needed, but doesn't add much complexity leaving the old variables in.

Reply:
OK, that was not a blocker anyways